topics/dev/tutorials/core-contributing/tutorial.md

```diff
@@ -298,3 +410,5 @@ component from the analysis router above by importing it from
 ``FavoriteExtensions``.
 
 {% include topics/dev/tutorials/core-contributing/index.js_diff.md %}
+
+# Conclusion
```

Comment:
Is there a conclusion expected down here?

Reply:
No additional paragraph text; I added it so that the key points section is displayed separately (and not as part of the previous section)